tests/serializers/test_json.py

```diff
@@ -92,6 +92,29 @@ def test_json_deserializer_exception(self):
             for obj in serializers.deserialize("json", """[{"pk":1}"""):
                 pass
 
+    def test_different_primary_key_name_and_attname_with_broken_natural_key(self):
+        """
+        If there is a primary key which is a foreign key it will have a different name
+        and attname (with `'_id'` suffix). In such case deserialization should work
+        fine, without relying on a natural key.
+        """
+        test_string = """[{
+            "pk": 1,
+            "model": "serializers.author",
+            "fields": {
+                "name": "Chuck Norris"
+            }
+        }, {
+            "pk": "1",
+            "model": "serializers.ghostwriter",
+            "fields": {
+            }
+        }]"""
+        dobjs = list(serializers.deserialize('json', test_string))
+        self.assertEqual(len(dobjs), 2)
+        for dobj in dobjs:
+            self.assertEqual(dobj.object.pk, 1)
```

Comment:
This is not a `JSON`-specific test, so can we move it to the `tests/serializers/test_natural.py`?

Reply:
Yes, it's not `JSON`-specific - makes perfect sense to move to `test_nautral.py`.